key_test.go

```diff
@@ -230,7 +231,7 @@ func Test_Key_UnmarshalCBOR(t *testing.T) {
 				0x01, 0x01, // kty: OKP
 				0x20, 0x01, // curve: CurveP256
 			},
-			WantErr:  "OKP curve must be X25519, X448, Ed25519, or Ed448; found \"P-256\"",
+			WantErr:  "Key type mismatch for curve \"P-256\" (must be EC2, found OKP)",
```

Comment:
_nit:_ here and in other places, use raw rather than interpreted string literal?